blank-canvas-blocks/sass/base/_alignment.scss

```diff
@@ -26,4 +26,19 @@
 // class which would do this for us.  I'm not sure why but this centers things appropriately.
 .aligncenter {
 	text-align: center;
-}
\ No newline at end of file
+}
+
+// These blocks go right to the edge of the viewport
+.wp-block-image,
+.wp-block-cover,
+.wp-block-video {
+	&.alignfull {
+		margin: 0 calc(-1 * var(--wp--custom--margin--body-horizontal));
+		width: calc(100% + var(--wp--custom--margin--body-horizontal)*2);
+
+		@include break-mobile {
+			margin: 0 calc(-1 * var(--wp--custom--margin--body-horizontal-desktop));
+			width: calc(100% + var(--wp--custom--margin--body-horizontal-desktop)*2);
+		}
```

Comment:
Redefining variables (as noted below in _body.scss) would eliminate the need for additional breakpoints like this.  There would just be one calculation, the value of the variable changing depending on the viewport width.